examples/ar_anchors/index.html

```diff
@@ -32,6 +32,17 @@
 				transform: translate(-50%, -50%);
 				font-size: 20px;
 			}
+			#description {
+			    pointer-events: auto;
+			    font-family: sans-serif;
+			    padding: 1em;
+			    background-color:rgba(255,255,255,0.7);
+			    -webkit-backdrop-filter: blur(5px);
```

Comment:
These samples need to work on more than just the WebXR Viewer, so please include the non-prefixed CSS.